src/db.js

```diff
@@ -204,82 +193,107 @@
 
                 return {
                     desc,
+                    distinct,
                     execute,
                     filter,
-                    distinct,
+                    limit,
                     map
                 };
             };
-            const filter = function (...args) {
-                filters.push(args.slice(0, 2));
+
+            const limit = function (...args) {
```

Comment:
Actually, upon further inspection, there actually had been a reason for the old way (it had checked for the length of the array, and if only 1 length, it added to it) but I've refactored `limit` to use named variables as well as `filter` which had similar checking.